scripts/collect-windows-logs.ps1

```diff
@@ -1,11 +1,24 @@
-$lockedFiles = "kubelet.err.log", "kubelet.log", "kubeproxy.log", "kubeproxy.err.log", "containerd.err.log", "containerd.log", "azure-vnet-telemetry.log"
+$lockedFiles = "kubelet.err.log", "kubelet.log", "kubeproxy.log", "kubeproxy.err.log", "containerd.err.log", "containerd.log", "azure-vnet-telemetry.log", "azure-vnet.log"

```

Comment:
Do you want to add azure-vnet-ipam.log here?

Reply:
sure why not. I haven't seen it fail on that file but if azure-vnet.log can be locked this one probably can too.

Reply:
interesting - I don't have a `c:\k\azure-vnet-ipam.log`

Reply:
just azure-vnet-ipam.json - is that what you meant?

Reply:
I took azure-vnet-ipam.log back out and rebased to keep the history clean

Reply:
strange